docs/source/examples/syncing-code-artifacts.rst

```diff
@@ -6,7 +6,7 @@ Syncing Code and Artifacts
 SkyPilot simplifies transferring code, data, and artifacts to and
 from cloud clusters:
 
-- To :ref:`upload code and project files <upload-code-and-project-files>` - use :code:`workdir`
+- To :ref:`upload code and project files or clone a git repository <upload-code-and-project-files-git>` - use :code:`workdir`
```

Comment:
how about `sync code and project files from a local directory or git repository`?